.github/workflows/publish-cdn.yml

```diff
@@ -22,7 +22,7 @@ jobs:
       - name: build:all
         run: npm run build:all
       - name: run tests
-        run: node ./tools/jil/bin/cli.js -f merged -b chrome@latest -s -t 85000 --concurrent=4
+        run: node ./tools/jil/bin/cli.js -f merged -b chrome@latest -s -t 85000 --concurrent=20
```

Comment:
Did you mean to leave this in here?

Reply:
Actually, i think we should leave this.  It could be helpful when its time to publish to CDN again to not have to wait 15 minutes for tests to run.